native/storybook/stories/index.js

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable import/no-extraneous-dependencies, import/no-unresolved, import/extensions */
```

Comment:
Can we move that actual stories to live inside component directories?

i.e. line 22 of this file could live in `native/packages/react-native-bpk-component-text/stories.js`

Reply:
Yep, that makes sense, will do that.

Reply:
Done